### PR TITLE
16637 Don't show numbered flow for CP + variable Incorporate Now button label

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#16637

*Description of changes:*

-  For Coop don't show numbered.
-  Re-use the existing feature flag for link to COLIN (until BC/CCC/ULC are released).
-  Verify remaining flow.
-  **Trying to add/fix unit tests with no luck yet**. Still looking into it
-  Use FF to determine whether to redirect to COLIN or Continue In Now

Temp url: https://namerequest-dev--pr-639-c9rqzr4l.web.app/

Some screenshots showing the Continuation In flow:

COOP:

image

image

Limited company Named:

image

image

**Approved NR Limited company Named**:

image

Limited company Numbered: Will always show link to Colin as Continuation In applications are not yet implemented 

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-639-c9rqzr4l.web.app

File: src/components/common/bullets-colin-link.vue
Comment (severinbeauvais) [inline src/components/common/bullets-colin-link.vue:177]: You could also just assign the appropriate Vuetify helper class (px-6) on line 21.

Ref: https://v2.vuetifyjs.com/en/styles/spacing/

File: src/components/new-request/search.vue
Comment (severinbeauvais) [inline src/components/new-request/search.vue:681]: Can you add a screenshot showing what this change does please?

**Comment by severinbeauvais** [comment]
When the NR is approved, what does the Existing NR page look like for your Continuation In examples above?

File: src/components/new-request/search.vue
Comment (ketaki-deodhar) [reply src/components/new-request/search.vue:681]: This is not used. I added it while playing with the 'Select action' input text to show 2 lines. I will remove this.

**Comment by ketaki-deodhar** [comment]
@severinbeauvais this is how approved NR looks. It does not show 'Incorporate Now' button.

Approved NR Limited company Named:

image

**Comment by severinbeauvais** [comment]
Thanks for the approved Continuation In NR. Should it should a link to COLIN?

**Comment by ketaki-deodhar** [comment]
> Thanks for the approved Continuation In NR. Should it should a link to COLIN?

I looked in the code and there aren't any instances we show COLIN link in this component but definitely can be added. Should I check with Yui/Janis?

**Comment by severinbeauvais** [comment]
> > Thanks for the approved Continuation In NR. Should it should a link to COLIN?
> 
> I looked in the code and there aren't any instances we show COLIN link in this component but definitely can be added. Should I check with Yui/Janis?

We just decided this morning: no button to COLIN. (Same thing we have now.)

File: src/components/common/bullets-colin-link.vue
Comment (ketaki-deodhar) [reply src/components/common/bullets-colin-link.vue]: removed this css and used px-6

**Comment by pwei1018** [comment]
Temporary Url for review: https://namerequest-dev--pr-639-c9rqzr4l.web.app

File: src/components/common/bullets-colin-link.vue
Comment (severinbeauvais) [reply src/components/common/bullets-colin-link.vue]: Fixed.

File: src/components/new-request/search.vue
Comment (severinbeauvais) [reply src/components/new-request/search.vue]: Fixed.

**Review by severinbeauvais — APPROVED** [review]
LGTM but please get another review or 2 before merging. (You could ask Travis to add their new dev to this PR.)

**Comment by ketaki-deodhar** [comment]
> LGTM but please get another review or 2 before merging. (You could ask Travis to add their new dev to this PR.)

as a reviewer?

**Comment by severinbeauvais** [comment]
> > LGTM but please get another review or 2 before merging. (You could ask Travis to add their new dev to this PR.)
> 
> as a reviewer?

Yes, but I'm mixed up. This is Namerequest. That other dev is working in Auth Web.

I'll ask Eve. She likes Namerequest I think :)

**Comment by ketaki-deodhar** [comment]
@seeker25 can you please add your new dev to this PR?

**Review by eve-git — APPROVED** [review]

**Comment by seeker25** [comment]
LGTM

**Pull request merged**
